chains/73773/chain-information.json

```diff
@@ -0,0 +1,48 @@
+{
+  "chainName": "Swimmer Network Subnet",
```

Comment:
Should this be "Swimmer Play Testnet"?

Reply:
This is fine - we're just keeping the same name that we had (I checked with Patrick)